pilot/cmd/pilot-agent/app/cmd.go

```diff
@@ -311,6 +317,73 @@ func initProxy(args []string) (*model.Proxy, error) {
 	return proxy, nil
 }
 
+func applyExcludeInterfaces(ifaces []string) []string {
+	// Get list of excluded interfaces from pod annotation
+	annotations, err := bootstrap.ReadPodAnnotations("")
+	if err != nil {
+		log.Infof("Failed to read pod annotations to find exluded interfaces. Continuing without exclusions. Proxy IPAddresses: %v", ifaces)
+		return ifaces
+	}
+	value, ok := annotations[annotation.SidecarTrafficExcludeInterfaces.Name]
+	if !ok {
+		log.Infof("excludeInterfaces annotation not present. Proxy IPAddresses: %v", ifaces)
+		return ifaces
+	}
+	exclusions := strings.Split(value, ",")
+
+	// Find IP addr of excluded interfaces and add to a map for instant lookup
+	exclusionMap := make(map[string]struct{})
+	for _, ifaceName := range exclusions {
+		iface, err := net.InterfaceByName(ifaceName)
+		if err != nil {
+			log.Warnf("Unable to get interface %s: %s", ifaceName, err.Error())
```

Comment:
```suggestion
			log.Warnf("Unable to get interface %s: %v", ifaceName, err)
```

here and below. Not sure it matters in practice but its what we do in the rest of the code